start_pokealarm.py

```diff
@@ -139,7 +145,13 @@ def check_for_update():
                         'data/shiny_data.json')
                         or sig['shiny_possible'] != shiny_possible_sig),
                     'invasions': (not os.path.isfile('data/invasions.json')
-                                  or sig['invasions'] != invasions_sig)
+                                  or sig['invasions'] != invasions_sig),
+                    'rankings': (
+                        not os.path.isfile('data/rankings-1500.json') or
+                        not os.path.isfile('data/rankings-2500.json') or
+                        not os.path.isfile('data/rankings-10000.json') or
```

Comment:
is 10k the max CP for any league?

Reply:
10k cp is just arbitrary and refers to Master league. This number comes from [source file names](https://github.com/pvpoke/pvpoke/tree/master/src/data/rankings/gobattleleague/overall).